### ISSUE TITLE
Any more development?

### ISSUE THREAD

**Issue by madasus** [sdague/waterfurnace#8]
* waterfurnace version:Series 7
* Python version: 3.7
* Operating System: Docker HA

### Description

Just curious - Is there likely to be any future development on this and the integration to home assistantwith respect to any of the following?

1. Supporting multiple units under a single symphony account
2. Supporting multiple stats on a single unit.  It looks like it currently just pulls the main stat
3. Adding any write-back to be able to change temp

Appreciate all the work that has gone into this project already!

M

**Comment by mdallaire** [comment]
I am also interested in having more features.

I know sdague did not want to add any write-back to the module so I guess someone else will have to do it and submit a PR. A quick search on github returned the following project that seems to have successfully implemented write-back to Symphony : https://github.com/cmilbert/waterfurnace-proxy/blob/master/waterfurnace-proxy.js

**Comment by sdague** [comment]
I might be able to swing around on this later. Sorry, it's been a pretty busy fall.

**Comment by madasus** [comment]
we have a zone controller on each of the units as well - it doesn't look like the wf-proxy is aware of the multi-zone units either.  I connect to the AWL locally and pull the machine level data - but that also doesn't seem to be able to provide me with any information about the zone controller.

I will have to get it from the websocket connection to Symphony.

**Comment by h2ouw8n4** [comment]
@sdague I've been itching to dig into this too so see if I could collaborate on this with you.  I have all the desire, but finding time is tough.  Hopefully, I can squeeze some time in somewhere.

**Comment by RyPeck** [comment]
I am particularly interested in suggestion #3 - I would like to alight my thermostats schedule with sunrise and sunset. I might have time at an upcoming Hackathon to dedicate some work @sdague

**Comment by RyPeck** [comment]
I am particularly interested in suggestion item 3 - I would like to align my thermostat programming with sunrise and sunset plus or minus so many minutes. I might have time at an upcoming Hackathon to dedicate some work if you are welcome to the features @sdague

**Comment by heffneil** [comment]
Yeah I have some issues like I can only see one unit (known issue apparently) and it appears that we can't adjust the thermostats like, well real thermostats?  

Having those features would be very nice.

**Comment by madasus** [comment]
my AWL died and I had to order a new one from WF.  The new one is running "pages" version 1.15 and i can no longer access the web-AID tool when it is connected to my WIFI.  Now all that I can access through WIFI is the Web Link status page (which is pretty useless).  Anyone else using the latest AWL figured out how to access (or bring back) the web AID tool access?

**Comment by heffneil** [comment]
How do you access it before?  You think they locked it down? I never tried it before so I am curious how to access it.  I was researching other thermostats to use that would integrate better with HA.

**Comment by madasus** [comment]
just type the IP address of your AWL into the browser and then select the Aid Tool option.  Previously I was just scraping the data there and using that to track what was going on with the units.  Unfortunately that menu option is gone now.

**Comment by heffneil** [comment]
Mine is 


Web Pages Version | 1.08

I don't have any links.  Some people were talking about URL variables to dance around.  I don't know if you have any we could try?  Are you on discord by any chance?

**Comment by madasus** [comment]
The old unit looks like this when you connect to it

![image](https://user-images.githubusercontent.com/9310056/130374737-826a43c5-983b-446a-9a6d-1bba818d4d87.png)

here is the link if you want to give it a try

http://IP/?pageid=at&params=2;300;;3761;127;103&units=0

On the new unit that link (and any other i try) takes me to the main page and there appears to be no other pages

![image](https://user-images.githubusercontent.com/9310056/130374772-5f82568e-7e50-40f9-9323-53ccbe78ecb4.png)

**Comment by heffneil** [comment]
AWL Version | AWLSTD 1.02
-- | --
Bootloader Version | AWLBLD 1.00



Not working for me either. There are other projects on github that connect to modbus and convert to mqtt.  I am curious to try them but I'm not sure I am technical enough to get it working but I will try.  I don't know if you looked at them.  Are you on the Home Assistant discord ever?

**Comment by madasus** [comment]
I did tap the modbus - it was interesting to play with but i didn't spend enough time to build out a full spec.  Unfortunately I found that with an Intellizone2 board I think you have to tap it in two places to get everything.  I was much happier using IP to the AWL - but now it looks like a websocket is the only option.

I'm not big into Home Assistant - still on Homeseer.  Home Assistant so far is just a test box.

**Comment by heffneil** [comment]
Im on the discord almost all the time if you want to chat real time.  

There are some devices that do USB to Modbus is that what you used?  The websocket up to Symphony isn't good - at all.  Its only one unit and you can't change anything only read.  Such a shame.  WaterFurnace isn't a tech company so its about as good as its going to get I think.

**Comment by madasus** [comment]
the websocket limitation is a limitation in this project - not on the WF end.  Using the WF websocket you can access both machines and modify data (thats how the web interface works)

**Comment by heffneil** [comment]
Yes - I realize that but Im not developing anything more and I don't know who to engage to do it ;)  Also I would prefer to access things locally so that I don't have to rely on the internet connection in the event of an outage...

**Comment by madasus** [comment]
lets keep in touch - now that i've lost my AWL direct ability i will be starting to look again.

**Comment by ccutrer** [comment]
FYI you might want to look at https://github.com/ccutrer/waterfurnace_aurora. it bypasses the AWL entirely, and talks (waterfurnace-specific variant of) modbus natively. I have full read/write control of my 7-series with an IZ2 and 6 zones. Also ticks the box of interfacing with multiple units (just run another instance of it). And no dependence on WaterFurnace's cloud service running, or the AWL dropping its wifi connection (my AWL is notoriously bad at that). Happy to add support for single-zone systems if people are willing to help me map the modbus registers out on your units. it publishes/listens to MQTT so is easy to integrate into home automation systems. I don't tweak my temperatures based on sunrise, but I do turn off zones if there are any windows/doors left open.

**Comment by madasus** [comment]
Cody - thank you! i'll dig up my rs-485 tap and cable I built and give this a try over the weekend
